app/script/components/contextMenu.coffee

```diff
@@ -0,0 +1,101 @@
+#
+# Wire
+# Copyright (C) 2016 Wire Swiss GmbH
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see http://www.gnu.org/licenses/.
+#
+
+window.z ?= {}
+z.components ?= {}
+
+z.components.ContextMenuEvent =
+  CONTEXT_MENU: 'customContextMenu'
```

Comment:
Usually we declare custom values like this:

```coffee
z.components.ContextMenuEvent =
  CONTEXT_MENU: 'z.components.ContextMenuEvent::CONTEXT_MENU'
```

With this we can copy the value `z.components.ContextMenuEvent::CONTEXT_MENU` and search through the code where this value is being used. Makes it easier to find it's reference.